build.yaml

```diff
@@ -621,6 +623,36 @@ steps:
     - build_hail
     - deploy_scheduler
     - deploy_executors
+ - kind: runImage
+   name: test_hail_java_distributed_backend
+   image:
+     valueFrom: hail_run_image.image
+   resources:
+     requests:
+       memory: "1G"
+       cpu: "1"
+   script: |
+     set -ex
+     cd /io
+     mkdir -p src/test
+     tar xzvf resources.tar.gz -C src/test
+     HAIL_TEST_SCHEDULER_HOST=scheduler.{{ default_ns.name }}.svc.cluster.local HAIL_TEST_SKIP_R=1 java -cp hail-test.jar:$SPARK_HOME/jars/* org.testng.TestNG -listener is.hail.LogTestListener testng-distributed-backend.xml
+   inputs:
+    - from: /resources.tar.gz
```

Comment:
I don't think you need the resources because you don't use them in the Distributed/Scheduler test.  Also, there's a local/global issue: they won't be accessible from the executors since they're in the test container.  They'll need to be copied into some service account's bucket eventually.